policies/templates/gcp_iam_allowed_policy_member_domains.yaml

```
@@ -81,14 +83,26 @@ spec:
             
             	members_to_check := [m | m = unique_members[_]; not starts_with_whitelisted_type(member_type_whitelist, m)]
             	member := members_to_check[_]
-            	matched_domains := [m | m = member; re_match(sprintf("[:@.]%v$", [params.domains[_]]), member)]
-            	count(matched_domains) == 0
+            	allow_sub_domains := lib.get_default(params, "allow_sub_domains", true)
+            	no_match(allow_sub_domains, params, member)
```

@morgante do you know how to pass just the `domains` to this function?

---

This should work, no?

```suggestion
            	no_match(allow_sub_domains, params.domains, member)
```

---

Thanks. It works. I was playing with the [] and _, and got confused.